openssl-dynamic/src/main/c/ssl_private.h

```diff
@@ -189,6 +189,10 @@ extern void *SSL_temp_keys[SSL_TMP_KEY_MAX];
 #define SSL_ERROR_WANT_PRIVATE_KEY_OPERATION -1
 #endif // SSL_ERROR_WANT_PRIVATE_KEY_OPERATION
 
+#ifndef SSL_MODE_ENABLE_FALSE_START
+#define SSL_MODE_ENABLE_FALSE_START 0x00000080L
```

Comment:
This fallback value is intend to be a no-op on OpenSSL I'm guessing? In that case, it probably shouldn't be `0x00000080L` or it may collide with other bits. Perhaps just zero?

Reply:
@davidben good idea... Also can you chime in on this one ?

https://github.com/netty/netty/pull/10407#issuecomment-658846738